inc/classes/subscriber/third-party/plugins/class-smush-subscriber.php

```diff
@@ -24,9 +26,14 @@ public static function get_subscribed_events() {
 			];
 		}
 
+		$prefix = rocket_get_constant( 'WP_SMUSH_PREFIX', 'wp-smush-' );
```

Comment:
this is used here and on line 104 too, maybe we can set a property in the constructor to hold this value?

Reply:
The benefit seems very low in terms of performance, it will require to do this on both places:
```php
if ( ! isset( self::$prefix ) ) {
    self::$prefix = rocket_get_constant( 'WP_SMUSH_PREFIX', 'wp-smush-' );
}
```

Reply:
no you're right, and since `get_subscribed_events` is static that doesn't work anyway